recipes/silvimetric/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "silvimetric" %}
+{% set version = "1.2.0" %}
+{% set sha256 = "1494ac17e9a13bc7eb3278847ccdcdb87ad9fdc55eb0b0c39ce7486aa7aab465" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/silvimetric-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [(win and vc<14) or py<310]
```

Comment:
@hobu do you believe this one can be noarch? If the reason to skip Windows is due to a missing dependency, we still prefer noarch to save resources and keep the build matrix small. Note that the final artifact won't be installable on Windows anyway and, if the missing dependency is ever built, this package will be ready for it.